src/main/resources/db/migration/V1_2__rename_certified_columns_to_approved_in_billingrecord_table.sql

```diff
@@ -0,0 +1,6 @@
+
+    alter table billing_record 
+    change column certified approved datetime(6);
+    
+    alter table billing_record
```

Comment:
Is this tested with the same version that runs on the db-machines? (10.3)

I know that we sometimes has pushed SQL-syntax that isn't supported in older versions.

Reply:
Yes, it's tested